gemrb/unhardcoded/bg1/colors.2da

```diff
@@ -5,3 +5,4 @@ TOOLTIP			0x00000000
 TOOLTIPBG		0xf0b080ff
 MAPICNBG		0xf0b080ff
 MAPTXTBG		0xf0b080ff
+ALTDOOR		    0xff00ffff
```

Comment:
The tables appear to use tabulators as separators, that's why the lines look misaligned in the web diff. Please match this in the new lines, so the columns will be nice and pretty.